README.md

```diff
@@ -170,6 +175,12 @@ charts:
     # --reset flag. It defaults to "0.0.1-set.by.chartpress". This is a valid
     # SemVer 2 version, which is required for a helm lint command to succeed.
     resetVersion: 1.2.3
+    # Use the version in Chart.yaml for the base version,
+    # instead of the latest tag from `git describe`
+    # This gives you more control over development version tags
+    # and lets you ensure prerlease tags are always sorted in the right order
```

Comment:
Typo in prerelease here. I'm not sure why but I can't comment a suggestion.

EDIT: oops okay, because I'm reviewing the changes associated with a specific commit, not the full PR.